app/src/main/java/cloudcity/networking/models/Iperf3NetworkDataModel.java

```diff
@@ -0,0 +1,79 @@
+package cloudcity.networking.models;
+
+import androidx.annotation.NonNull;
+
+import com.google.gson.annotations.SerializedName;
+
+import cloudcity.dataholders.MetricsPOJO;
+import de.fraunhofer.fokus.OpenMobileNetworkToolkit.DataProvider.LocationInformation;
+
+public class Iperf3NetworkDataModel extends NetworkDataModel {
+
+    @SerializedName("category")
+    final private String category = "Iperf3";
+
+    @SerializedName("lat")
+    private double latitude;
+    @SerializedName("lon")
+    private double longitude;
+
+    @SerializedName("accuracy")
+    private double accuracy;
+    @SerializedName("speed")
+    private double speed;
+
+    @SerializedName("values")
+    final private Iperf3ValuesModel values;
```

Comment:
this one as well, and make a `NetworkDataModelValues` 'marker' interface that they all have to implement (if possible and doesn't completely confuse retrofit; but it shouldn't i think/hope)

this would all be handled by GSON in the end and it won't get confused. most likely...

Reply:
done, and yes, GSON didn't have any problem with that.

Reply:
resolving.